luthier/config/src/config.lua

```diff
@@ -30,7 +30,14 @@ local tableConfig = tableUtils.build_table_config(tables)
 -- a script since creating a dir in Lua is awkard, in future.
 -- can be circumvented by using the LuaFileSystem module
 os.execute("mkdir -p ../external/")
+os.execute("mkdir -p ../../src/test/resource/")
```

Comment:
Should be `resources` not `resource` (note the plural). Also, I would
pull the string "../../src/test/resources/" into a local variable. That way
you can do:
```lua
os.execute("mkdir -p " . resourcesDirectory)
parser.save(resourcesDirectory . "DimensionConfig.json")
...
```
Makes it easier to change the resourcesDirectory if we need to,
and makes things a bit more readable in my opinion.

Reply:
That is a good suggestion.